contracts/VaultV1.sol

```diff
@@ -12,51 +12,35 @@ contract VaultV1 is ERC4626, Ownable {
     using SafeTransferLib for ERC20;
 
     //the underlying token the vault accepts
-    ERC20 public immutable UNDERLYING;
+    ERC20 public immutable underlying;
     //the strategy address
-    IStrategy public STRATEGY;
-    //the buffer amount (indicates to bot the minimun amount of tokens that can be stored in the vault)
-    uint256 public BUFFER;
+    IStrategy public strategy;
 
     // Withdraw locking params
     mapping(address => uint256) public requestedWithdraws;
     mapping(address => uint256) public unlockTime;
-    uint256 public totalRequestedWithdraws;
     uint256 public withdrawWindow = 24 hours;
 
-    constructor(ERC20 underlying, address strategy) ERC4626(underlying, "TracerVault", "TVLT") {
-        UNDERLYING = ERC20(underlying);
-        STRATEGY = IStrategy(strategy);
+    constructor(ERC20 _underlying, address _strategy) ERC4626(_underlying, "TracerVault", "TVLT") {
+        underlying = ERC20(_underlying);
+        strategy = IStrategy(_strategy);
```

Comment:
Vault requires strategy in constructor, but strategy also requires vault in constructor. Should we use an instantiate method?